mlrun/model_monitoring/controller.py

```diff
@@ -633,7 +723,15 @@ def handler(context: nuclio_sdk.Context, event: nuclio_sdk.Event) -> None:
 
     if event.trigger.kind == "http":
         # Runs controller chief:
-        MonitoringApplicationController().push_regular_event_to_controller_stream(event)
+        context.user_data.monitor_app_controller.push_regular_event_to_controller_stream(
+            event
```

Comment:
+ remove it from the func signature (not in used). Currently we don't need the event data for the chief. 
```suggestion
```